packages/css/test/index.ts

```diff
@@ -6,6 +6,17 @@ const theme: Theme = {
     secondary: 'cyan',
     background: 'white',
     text: 'black',
+    colorScale: {
+      default: 'color500',
+      100: 'color100',
```

Comment:
Nit: I'd prefer to see valid CSS color values here -- I think it helps with reading the tests and making sure it's doing what you'd expect to happen